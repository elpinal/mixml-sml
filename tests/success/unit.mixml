link U =
  {module U =
    [unit
      {module u = [type :Type]}
    ]
  }
with
  {module V = [unit {}]}
