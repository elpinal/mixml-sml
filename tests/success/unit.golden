{U : [forall (!4@u : Type). {u : [= !4]}]+, V : [{}]+}
